TicketBooking/UserInterface/ManagerInterface.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TicketBooking.Class;
+using TicketBooking.Flight.FlightModel;
+using TicketBooking.Flight.FlightRepository;
+using TicketBooking.User.Passenger.Bookings;
+using TicketBooking.User.Passenger.PassengerModel;
+using TicketBooking.User.UserRepository;
+
+namespace TicketBooking.UserInterface
+{
+    public class ManagerInterface
+    {
+        FlightRepository _flights = new FlightRepository();
+        List<BookingsModel> _bookings = new List<BookingsModel>();
+
+        public ManagerInterface()
+        {
+            UserRepository users = new UserRepository();
```

Comment:
Please let the consrtuctor accept `UserRepository`. 
Same for `booking`, `flights`. variables

Dependancy inversion.